include/zephyr/bluetooth/audio/bap.h

```diff
@@ -317,6 +317,25 @@ struct bt_bap_scan_delegator_recv_state {
 };
 
 struct bt_bap_scan_delegator_cb {
+
+	/**
+	 * @brief Attempt to add a new source to a receive state failed
+	 *
+	 * An attempt to add a new broadcast source to a receive state failed.
+	 *
+	 * @param conn       Pointer to the connection to a remote device if
+	 *                   the source was added from it, otherwise NULL.
+	 * @param recv_state Receive state that was attempted to be added.
+	 * @param err        errno explaining why adding the source failed.
+	 *
+	 * return true if the intended operation of the callback was successful,
+	 *        or false if unsuccessful
+	 *
+	 */
+	bool (*recv_state_add_src_fail)(struct bt_conn *conn,
+					struct bt_bap_scan_delegator_recv_state recv_state,
```

Comment:
```suggestion
					const struct bt_bap_scan_delegator_recv_state *recv_state,
```

Reply:
I agree, but my point here was that you can just provide a `const` pointer to the copy, rather than passing it by value (minor optimization)